src/components/Interview/Interview.tsx

```diff
@@ -35,13 +40,34 @@ interface InterviewProps {
     rejectReasons: RejectReason[];
 }
 
+const StyledTitle = styled(Text)`
+    margin-top: 50px;
+    display: flex;
+    margin-bottom: 40px;
+    cursor: pointer;
```

Comment:
Very strange styles for title. Is it copypasted?

Reply:
oh, yes, I deleted it